.github/workflows/regen_certs.yml

```diff
@@ -0,0 +1,35 @@
+name: Regenerate WPT certificates
+on:
+  schedule:
+    # Our certificates are good for ~2 years, but we regenerate them every
+    # month to be sure.
+    - cron: '0 0 1 * *'
+jobs:
+  update:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+    - name: Regenerate certs
+      # Use a conditional step instead of a conditional job to work around #20700.
+      if: github.repository == 'web-platform-tests/wpt'
+      run: |
+        python wpt make-hosts-file | sudo tee -a /etc/hosts
+        python wpt regen-certs --force
+    - name: Commit and create pull request
+      # Use a conditional step instead of a conditional job to work around #20700.
+      if: github.repository == 'web-platform-tests/wpt'
+      uses: peter-evans/create-pull-request@v2
+      with:
+        token: ${{ secrets.GITHUB_TOKEN }}
+        commit-message: Automated regeneration of WPT certificates
+        title: Regenerate WPT certificates
+        body: |
+          This automated pull request updates the core WPT certificates.
+
+          Please test the new certs locally before merging this pull request.
```

Comment:
What manual testing is require here? I'd expect the infrastructure tests to break if the cert is invalid. Asking people to do unspecified work doesn't seem like the best idea.

Reply:
Yeah. Is running infrastructure/ tests enough? If we're worried about SXG, we can include one SXG test there.